### PR TITLE
Open applications

### PR DESCRIPTION
**Pull request by minaorangina**
also renames some files and removes the "workshops for women" bit in the title

### CODE REVIEW

File: chapters.html [hunk 1]
```diff
@@ -122,11 +122,5 @@ <h5 class="white-text">Contact Us</h5>
       </div>
     </div>
   </footer>
-
-  <!--  Scripts-->
-  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
```
Comment (claireinez) [inline chapters.html:127]: No scripts for this page?

File: js/all-events-template.js [hunk 2]
```diff
@@ -93,7 +74,7 @@ function generateApplicationText (event) {
   if (moment(event.date).isAfter(TODAY))  {
     text = '<span class="application-text">' + event.application_text + '</span>';
     if (event.application_link && event.application_link.length > 0) {
-      text = '<a href="' + event.application_link + '">' + text + '</a>';
+      text = '<a href="' + event.application_link + '" target="_blank">' + text + '</a>';
```
Comment (claireinez) [inline js/all-events-template.js]: Not the biggest fan of `target="_blank`. Do we need it here?

**Review by claireinez — APPROVED** [review]

**Comment by msmichellegar** [comment]
(Soz Mina just decided to merge @minaorangina)

**Comment by minaorangina** [comment]
Thanks @msmichellegar, was barely at my computer yesterday so didn't see it was approved!

**Pull request merged**